hazelcast/src/main/java/com/hazelcast/cache/impl/CacheDataSerializerHook.java

```diff
@@ -105,9 +106,10 @@
     public static final short REMOVE_ALL = 34;
     public static final short REMOVE_ALL_BACKUP = 35;
     public static final short REMOVE_ALL_FACTORY = 36;
-    public static final short MERGE = 37;
+    public static final short PUT_ALL = 37;
```

Comment:
No specific reason. just to see `PUT_ALL` after `REMOVE_ALL` :)
This is for 3.6 release and won't be backported, so I don't think there will be a backward compatibility issue.